examples/components-macos/app/project.yml

```diff
@@ -17,25 +19,25 @@ targets:
       - Resources
       - Sources
     settings:
-      HEADER_SEARCH_PATHS: '$(STDLIB_PATH)'
+      HEADER_SEARCH_PATHS: '$(STDLIB_PATH) $(HEADER_SEARCH_PATHS)'
       LIBRARY_SEARCH_PATHS: '$(STDLIB_PATH)'
-      OTHER_LDFLAGS: ' -framework Cocoa'
+      OTHER_LDFLAGS: '$(OTHER_LDFLAGS)'
```

Comment:
oh yeah, I think we should just pass `-framework Cocoa` here - to have the OCaml related things in dune files and the Xcode related things in project.yml